packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -664,30 +686,6 @@ class DropdownCore extends React.Component<Props, State> {
                 ? this.state.itemRefs[focusIndex].ref
                 : null;
 
-            // Render the SearchField component.
```

Comment:
I've simplified a lot how the `SearchField` is rendered (now it's completely separate from the `listbox` logic.